vscode/src/completion.ts

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
This and a few other files in the folder don't have the MIT header.

Also, what vars are unused and are they necessary?

Reply:
Some VS Code interfaces (e.g. `provideCompletionItems`) take a `CancellationToken` and other optional parameters I don't use, but might later. I'll move this comment below to just suppress that line.